equinix/resource_fabric_cloud_router_acc_test.go

```diff
@@ -3,103 +3,125 @@ package equinix
 import (
 	"context"
 	"fmt"
-	"testing"
-
-	"github.com/equinix/terraform-provider-equinix/internal/config"
-
 	v4 "github.com/equinix-labs/fabric-go/fabric/v4"
+	"github.com/equinix/terraform-provider-equinix/internal/config"
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
 	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"testing"
 )
 
-func TestAccCloudRouterCreate(t *testing.T) {
+func TestAccCloudRouterCreateOnlyRequiredParameters_PFCR(t *testing.T) {
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:     func() { testAccPreCheck(t) },
 		Providers:    testAccProviders,
 		CheckDestroy: checkCloudRouterDelete,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccCloudRouterCreateConfig("fg_tf_acc_test"),
+				Config: testAccCloudRouterCreateOnlyRequiredParameterConfig("fcr_tf_acc_test"),
```

Comment:
Same note about resource creation names needing to have the suffix of the account they are apart of for sweeping purposes. Use PFCR as the suffix.